L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCClusters.cc

```diff
@@ -26,6 +26,7 @@ class HGCalTriggerNtupleHGCClusters : public HGCalTriggerNtupleBase
     edm::EDGetToken clusters_token_, multiclusters_token_;
     HGCalTriggerTools triggerTools_;
 
+    TString prefix_;
```

Comment:
Better to use `std::string`

Reply:
Yes.. I would normally use std::string too, but because std::string doesn't have casting method `operator const char*()` like TString does, I'd have to write a bunch of `.c_str()` in the `Branch` lines, which makes the code quite ugly. I'll do it anyway though